src/trainer.py

```diff
@@ -101,8 +99,15 @@ def evaluate(self, loss_f, metric_fs={}, mode='dev'):
             metric = fn(binary_pred, mask)
             epoch_metrics[name] += metric
 
-    return (epoch_loss / n), {name: value/n for name, value in epoch_metrics.items()}
-  
+        if self.config.store_images and i % 10  == 0:
```

Comment:
maybe (10) should belong to the config file too, but doesn't matter now

Reply:
Sure, we can create the issue. There are only 48 dev images, so this didn't seem like such an important parameter.